backend/config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Spree::Core::Engine.add_routes do
   namespace :admin do
+    post '/addresses/shipment/:shipment_id', to: "addresses#create_shipment", as: :create_shipment_address
```

Comment:
I think I'd rather have the controller action a member of the shipment controller rather than the address controller.